src/platform/brand-consolidation/feature-flag.js

```diff
@@ -0,0 +1,8 @@
+/**
+ * Returns whether the current Webpack build is executing as the brand-consolidation of Vets.gov/VA.gov.
+ * @returns {boolean}
+ * @module platform/brand-consolidation/feature-flag
+ */
+export default function isBrandConsolidationEnabled() {
+  return process.env.BRAND_CONSOLIDATION_ENABLED !== undefined;
```

Comment:
It might be safer to force the variable to be true or false in the Webpack define plugin config, so that you don't have to do `!== undefined` in the application code and can treat it like a normal boolean.